regen-1/gentxs/gentx-b3929bf6504c5a87955d171158d5c338737d7e98.json

```diff
@@ -1 +0,0 @@
-{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Chainflow","identity":"","website":"","security_contact":"","details":"Independent Validator Providing Fair and Equitable Staking for All #KeepStakeDecentralized https://chainflow.io/staking"},"commission":{"rate":"0.080000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"regen1snn4uhxh04gzpgk4l8naw3n6fu7ucwx3zfv0hn","validator_address":"regenvaloper1snn4uhxh04gzpgk4l8naw3n6fu7ucwx37w320z","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"4ZO9Oj2esqGYQA6KiND6MYa8X/TCq3Xc4zFi1wytSHE="},"value":{"denom":"uregen","amount":"1000000000"}}],"memo":"","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AibuN5foIxPrciwIokFyGyAX3BGkUY6La/XEVUTU3Pcf"},"mode_info":{"single":{"mode":"SIGN_MODE_LEGACY_AMINO_JSON"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["MFNDKzTe9wXI+s8ZR8XornsnvNYFi7F8vY8m8klkXmY87I8PTCAmdBEnL3AvMEH593PEUY909xbsOhhBqUp1lg=="]}
```

Comment:
Why was this deleted?

Reply:
Nevermind I see

Reply:
@aaronc the gentx is invalid: https://github.com/regen-network/mainnet/pull/114#issuecomment-817932383